pyseir/load_data.py

```diff
@@ -654,7 +696,7 @@ def get_average_dwell_time(category):
     # TODO: This value is temporarily in this limited scope.
     # Will be added into the params once I decide on some refactoring.
     params["hospitalization_length_of_stay_icu_avg"] = 8.6
-    if category == "hospitalized":
+    if str(category) == "hospitalized":
```

Comment:
embrace the enum by avoiding str operations: `category == HospitalizationCategory.HOSPITALIZED:`

Reply:
fixed in the latest commit!